testing/public-api-tests/src/integTest/groovy/org/gradle/api/PublicApiIntegrationTest.groovy

```diff
@@ -139,6 +139,9 @@ class PublicApiIntegrationTest extends AbstractIntegrationSpec implements JavaTo
             import org.gradle.api.Plugin
             import org.gradle.api.Project
 
+            // Test if extension methods are available
+            import org.gradle.kotlin.dsl.the
```

Comment:
❓ Is it enough to just import it or we have to use it?

Reply:
I've added a better test for extension methods.